public/contents/6-advanced-concepts/tours.json

```diff
@@ -48,6 +48,25 @@
       "kuid": "f9e5f0t5-a8a5-4arb-bfu1-50d392n4",
       "hikeVersion": "1.0",
       "weight": "2"
+    },
```

Comment:
In tour.json, only change should be checksum for respective hike.
@AshokMamidiHcl please guide here